### PR TITLE
[8.0][MIG] product_price_history

### PR DESCRIPTION
**Pull request by astirpe**
This is the backport of `product_price_history` from V12 to V8

### CODE REVIEW

**Comment by astirpe** [comment]
@eLBati @marcelofrare @hbrunn @andreampiovesana @ferranmateu

File: product_price_history/README.rst [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/product-attribute-12-0/product-attribute-12-0-product_price_history
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/135/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
+
+This module allows to manage product history prices.
```
Comment (StefanRijnhart) [inline product_price_history/README.rst:28]: I know you just ported this "as is", but I think this description is a little lacking. It should mention something like "adds views for the existing functionality of product price history in code Odoo, and allows the user to access them using a smart button on the product form"

Comment (astirpe) [reply product_price_history/README.rst:28]: @StefanRijnhart indeed the description was needed. Added now. Thanks!

**Review by hbrunn — APPROVED** [review]

**Review by StefanRijnhart — APPROVED** [review]
Thanks!

**Comment by astirpe** [comment]
Could this one be merged?

**Comment by hbrunn** [comment]
could you add a test that looks at the code for the history count?

**Review by hbrunn — APPROVED** [review]

**Comment by StefanRijnhart** [comment]
Must be some module here that makes the default_code field required
```
2021-05-11 12:06:33,835 23275 ERROR openerp_test openerp.addons.product_price_history.tests.test_product_price_history: ` IntegrityError: null value in column "default_code" violates not-null constraint
```

**Comment by astirpe** [comment]
OK tests seem passing this way. Squashing last commit...

**Pull request closed without merging**